.devcontainer/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/devcontainers/javascript-node
```

Can we pin this image to a specific node version and probably reference a semantic version ? 
This way we could avoid inconsistencies, and build failures due to future (even breaking) updates to this image.

```suggestion
FROM mcr.microsoft.com/devcontainers/javascript-node:0-18
```